### DISCUSSION TITLE
Spam making it through to inbox

### DISCUSSION THREAD

**Discussion by knynkwl** [solspace/craft-freeform#764] [Questions & Answers] [answered]
Hey, we are using v3.11.9 and we are noticing that sometimes a spam email will come through and not be sent to the span folder. 

Currently, we have two levels of spam protection, 1. a honey pot and 2. Recaptcha v2

Are there any other ways of preventing spam from coming through? Could this be a issue with emails directly sending a POST request instead of going through the form? 

Does Freeform have some protection again this or would this need to be setup manually by a developer?

**Comment by kjmartens** [comment, marked as answer]
Hi @knynkwl,

Would you be able to update to Freeform 3.13.32? There have been many fixes and improvements pertaining to spam protection since 3.11.9. 🙂 
https://docs.solspace.com/craft/freeform/v3/setup/changelog/

**Reply by knynkwl** [reply]
We were able to update and seem to be working now. Thanks
